frontend/src/users/hooks/auth/useAuthorization.ts

```diff
@@ -1,22 +1,22 @@
 import jwt_decode from 'jwt-decode';
-import { AuthorizationConsts } from '@users/enums/authorization-consts';
-import { TimeUnit } from '@shared/enums/time-unit';
 import { useEffect } from 'react';
+import { TimeUnit } from '@shared/enums/time-unit';
+import { useJwt } from '@users/contexts';
 
 interface DecodedJwt {
   sub: string;
   exp: number;
 }
 
 export const useAuthorization = (): void => {
```

Comment:
Wydaje mi sie, ze teraz to powinnismy miec w `UserProvider`, a nie w `App.ts`. Wtedy cala logika zarzadzania userem bedzie w jednym miejscu. Wlasciwie bedzie mozna wywalic ten hook, i do `UserProvider` wkleic tylko 

```
useEffect(() => {
    if (!jwt) return;

    if (!isTokenExpired(jwt)) return removeJwt();
  }, []);
```

A to
```
 const isTokenExpired = (token: string): boolean => {
    const decodedJwt: DecodedJwt = jwt_decode(token);
    return decodedJwt.exp * TimeUnit.SECOND - TimeUnit.MINUTE > Date.now();
  };
```

do osobnego pliku trzeba